test/unit/annotation_spec.js

```diff
@@ -233,6 +233,74 @@ describe('annotation', function() {
     });
   });
 
+  describe('TextAnnotation', function() {
+    it('should not parse state model when not defined',
+        function (done) {
+      const annotationDict = new Dict();
+      annotationDict.set('Type', Name.get('Annot'));
+      annotationDict.set('Subtype', Name.get('Text'));
+      annotationDict.set('Contents', 'TestText');
+      const annotationRef = new Ref(819, 0);
+
+      const replyDict = new Dict();
+      replyDict.set('Type', Name.get('Annot'));
+      replyDict.set('Subtype', Name.get('Text'));
+      replyDict.set('IRT', annotationDict);
+      replyDict.set('RT', Name.get('R'));
+      replyDict.set('Contents', 'ReplyText');
+
```

Comment:
Remove the newline here, just like above, for consistency (applies to other blocks in tests below too since the newlines are a bit inconsistent throughout these changes).

Reply:
You're right, it is weird this way … so I tried to follow this pattern (I found it in other tests in this file and I think it is more readable)

```
Dict1

Dict2

Ref1
Ref2
xRef

ResultCheck
```

In one occasion, where I needed Ref1 before Dict2 I went with this pattern
```
Ref1
Dict1

Ref2
Dict2

xRef

ResultCheck
```
If you'd like to see it formatted some other way, please let me know :).

Reply:
Looks really good this way!